templates/fine-tune-llm_v2/cookbooks/long_context/README.ipynb

```diff
@@ -0,0 +1,115 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Fine-tuning on datasets with long context\n",
```

Comment:
This cookbook is not about extending the context though and I have avoided that wording everywhere afaics. You mention in your review:
> I just read this cookbook and I am lost about what the exact focus is.
>  Is it context length extension?
It's not context-length extension or as your comment suggests here ("extending context length"). I only mention extention in the FAQ because of that. We have always handled context-length extension as something that should be transparent to the user. We can tell them that quality suffers the more they exceed the native context length and that we use rope-scaling, I agree with that! But that's literally one sentence.

The focus is how to prepare your dataset if you want to fine-tune with context lengths that are so long that users worry about them. When you say "All user needs to do is to make sure their dataset is properly formatted at longer context" - I think that's THE part in a users journey where they might need some help. 

If I understand you correctly, we want to scrap this part, where we show how to chip away parts of your dataset that are too long. Instead, we should demonstrate choosing a dataset that has a context length that exceeds the native context length of a mode. With that in mind, I think we can scrap this cookbook alltogether and just make this a one-liner the main readme.